api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -3,8 +3,7 @@
 import static com.google.common.truth.Truth.assertThat;
 import static org.mockito.Matchers.any;
 import static org.mockito.Matchers.eq;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Per slack discussion, please avoid `*` imports. (we should turn on a linter rule for this)